canonical-websites/www.ubuntu.com
# Redirect index.html and *.html

## Done

* Redirect <page>.html to <page>
* Redirect index.html to /

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Go to [http://0.0.0.0:8001/index.html](http://0.0.0.0:8001/index.html)
- Check that the address is redirected to [http://0.0.0.0:8001/](http://0.0.0.0:8001/)


## Issue / Card

Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/1804
